changelog/2.069.0.dd

```diff
@@ -53,6 +53,14 @@ $(LI $(LNAME2 property-switch-deprecated, The $(TT -property) switch has
     $(P Since the behaviour of the $(TT -property) switch was not well-liked,
         it's been deprecated and made to have no effect when used.
     )
+
+    $(P The $(A $(ROOT_DIR)function.html#property-functions,$(D @property)
+        attribute) has not been deprecated. But since it only had an effect with
+        the $(TT -property) switch, it has no effect anymore. The future of
+        $(D @property) is uncertain. It's clear that there is a desire for
```

Comment:
I think an explanation why we're keeping a nop attribute around is in order. I don't know how to give a more positive spin.